frontend/src/screens/channels/IncreaseIncomingCapacity.tsx

```diff
@@ -203,246 +212,265 @@ function NewChannelInternal({
     <>
       <AppHeader
         title="Open Channel with Lightning"
-        description="Purchase a channel that allows you to receive payments"
+        description="Increase your receive limit by opening a new lightning channel for a small fee."
         contentRight={
           <div className="flex items-end">
             <Link
               to="/channels/outgoing"
               className="underline break-words text-sm"
             >
-              Open Channel with On-Chain
+              Open Channel with On-chain
             </Link>
           </div>
         }
       />
-      <div className="md:max-w-md max-w-full flex flex-col gap-5 flex-1">
-        <img
-          src="/images/illustrations/lightning-network-dark.svg"
-          className="w-full hidden dark:block"
-        />
-        <img
-          src="/images/illustrations/lightning-network-light.svg"
-          className="w-full dark:hidden"
-        />
-        <p className="text-muted-foreground">
-          Alby Hub works with selected service providers (LSPs) which provide
-          the best network connectivity and liquidity to receive payments. The
-          channel typically stays open as long as there is usage.{" "}
-          <ExternalLink
-            className="underline"
-            to="https://guides.getalby.com/user-guide/alby-account-and-browser-extension/alby-hub/faq-alby-hub/how-to-open-a-channel"
-          >
-            Learn more
-          </ExternalLink>
-        </p>
-        <form onSubmit={onSubmit} className="flex flex-col gap-5 flex-1">
-          <div className="grid gap-1.5">
-            <TooltipProvider>
-              <Tooltip>
-                <TooltipTrigger type="button">
-                  <div className="flex flex-row gap-2 items-center justify-start text-sm">
-                    <Label htmlFor="amount">
-                      Increase receiving capacity (sats)
-                    </Label>
-                    <InfoIcon className="h-4 w-4 shrink-0 text-muted-foreground" />
-                  </div>
-                </TooltipTrigger>
-                <TooltipContent className="w-[300px]">
-                  Configure the amount of receiving capacity you need. You will
-                  only pay for the liquidity fee which will be shown in the next
-                  step.
-                </TooltipContent>
-              </Tooltip>
-            </TooltipProvider>
+      <MempoolAlert />
```

Comment:
This is shown in the wrong place compared to in master

Reply:
resolved